backend/functions/lib/controllers/case-summary/case-summary.controller.test.ts

```diff
@@ -1,40 +1,46 @@
 import { createMockApplicationContext } from '../../testing/testing-utilities';
-import { CaseSummaryController } from './case-summary.controller';
 import CaseManagement from '../../use-cases/case-management';
 import { NotFoundError } from '../../common-errors/not-found-error';
 import { MockData } from '../../../../../common/src/cams/test-utilities/mock-data';
+import { mockCamsHttpRequest } from '../../testing/mock-data/cams-http-request-helper';
+import { CaseSummaryController } from './case-summary.controller';
 
 describe('Test case-summary controller', () => {
   let applicationContext;
   let controller;
 
   beforeEach(async () => {
-    applicationContext = await createMockApplicationContext({ DATABASE_MOCK: 'true' });
+    applicationContext = await createMockApplicationContext();
     controller = new CaseSummaryController(applicationContext);
   });
 
   test('should return success if case summary is found', async () => {
     const caseDetail = MockData.getCaseDetail();
 
     jest.spyOn(CaseManagement.prototype, 'getCaseSummary').mockResolvedValue(caseDetail);
-    const response = await controller.getCaseSummary(applicationContext, caseDetail.caseId);
-    expect(response).toEqual({ success: true, body: caseDetail });
+    const request = mockCamsHttpRequest({ params: { caseId: caseDetail.caseId } });
+    const response = await controller.getCaseSummary(applicationContext, request);
```

Comment:
again here.  The request should be in the context.

Reply:
Yes, but in this case the requests are overiden from the default request we would get back from the application context. Not changing.